docs/rtd-requirements.txt

```diff
@@ -0,0 +1,11 @@
+# force upgrades for rtd default packages https://git.io/JU73V
+setuptools~=50.3
+alabaster~=0.7.12
+commonmark~=0.9.1
+docutils~=0.16.0
+mock~=4.0
+pillow~=7.2
+readthedocs-sphinx-ext~=2.1
+recommonmark~=0.6.0
+sphinx~=3.2
+sphinx-rtd-theme~=0.5.0
```

Comment:
I like semver ranges, but I know others prefer unlimited version ranges.

also should these be in a `[docs]` extra? I don't want to put setuptools in an extra though